app/controllers/api/v2/infrastructures_controller.rb

```diff
@@ -25,6 +25,69 @@ def profile_index
     render json: profiles
   end
 
+  def helm_infrastructure
```

Comment:
if we see other apis, the convention will be `helm_infrastructure_by_cluster_name` like this [one](https://github.com/BaritoLog/BaritoMarket/blob/master/app/controllers/api/v2/infrastructures_controller.rb#L28)

Reply:
updated